plog/plog

```diff
@@ -542,7 +574,7 @@ main () {
 
         # If log contains multicast data, rename to _full.* since we will upload a reduced file
         FULL=
-        if zegrep -q "$PLOG_REDUCE_PATTERN" $log; then
+        if "$zegrep" -q "$PLOG_REDUCE_PATTERN" $log; then
```

Comment:
Maybe zegrep should be it's own function too (like compress and uncompress)

Reply:
Right. Since it used in two places with different forms (the other is in `reduce`),  I didn't quite see how to  do that with one function and preserve the `-t` functionality. Maybe it requires two functions, but that is also a little unattractive  I will try to make a second pass at it. I am no wiz with _bash_.  Suggestions would be welcome.

Reply:
Mmm yeah good point. My think was just follow the pattern of the others, but no point making it move complex than it needs to be.

Reply:
I change to a more unique name, from `zegrep` to `egrep_program`